### PR TITLE
Fix for Interop Json and Added new argument

### PR DESCRIPTION
**Pull request by mwaykole**
Signed-off-by: mwaykole <mwaykole@redhat.com>

### CODE REVIEW

**Comment by mwaykole** [comment]
Verified all Scenarios 

(venv)  ✘ mwaykole@m  ~/ods-ci 
 ⏩python3 utils/scripts/ocm/ocm.py   get_latest_osd_candidate_json  --json-path int.json --osd-major-version 4  --osd-minor-version-start 8  --osd-minor-version-end 12 --new_run=True
ods-ci-logger: INFO     {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}}
ods-ci-logger: INFO     True
ods-ci-logger: INFO     File is updated to : {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}, 'RUN': []} 
(venv)  mwaykole@m  ~/ods-ci 
   ⏩python3 utils/scripts/ocm/ocm.py   get_latest_osd_candidate_json  --json-path int.json --osd-major-version 4  --osd-minor-version-start 8  --osd-minor-version-end 12 --new_run=False
ods-ci-logger: INFO     {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}}
ods-ci-logger: INFO     False
ods-ci-logger: INFO     File is updated to : {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}, 'RUN': ['4.10-latest', '4.11-latest']} 
(venv)  mwaykole@m  ~/ods-ci 
  ⏩python3 utils/scripts/ocm/ocm.py   get_latest_osd_candidate_json  --json-path int.json --osd-major-version 4  --osd-minor-version-start 8  --osd-minor-version-end 12 --new_run=False
ods-ci-logger: INFO     {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}}
ods-ci-logger: INFO     False
ods-ci-logger: INFO     File is updated to : {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}, 'RUN': ['4.10-latest', '4.11-latest']} 
(venv)  mwaykole@m  ~/ods-ci 
   ⏩python3 utils/scripts/ocm/ocm.py   get_latest_osd_candidate_json  --json-path int.json --osd-major-version 4  --osd-minor-version-start 8  --osd-minor-version-end 12 --new_run=True 

ods-ci-logger: INFO     {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}}
ods-ci-logger: INFO     True
ods-ci-logger: INFO     File is updated to : {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}, 'RUN': []} 
(venv)  mwaykole@m  ~/ods-ci 
   ⏩python3 utils/scripts/ocm/ocm.py   get_latest_osd_candidate_json  --json-path int.json --osd-major-version 5  --osd-minor-version-start 8  --osd-minor-version-end 12 --new_run=False
ods-ci-logger: INFO     {}
ods-ci-logger: INFO     False
ods-ci-logger: INFO     File is updated to : {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}, 'RUN': []} 
(venv)  mwaykole@m  ~/ods-ci 
 ⏩python3 utils/scripts/ocm/ocm.py   get_latest_osd_candidate_json  --json-path int.json --osd-major-version 5  --osd-minor-version-start 8  --osd-minor-version-end 12 --new_run=False
ods-ci-logger: INFO     dict_keys(['4', 'RUN', '5'])
0:   presented only in oldods-ci-logger: INFO     False
ods-ci-logger: INFO     File is updated to : {'4': {'4.8': '4.8.46', '4.9': '4.9.40', '4.10': '4.10.21', '4.11': '4.11.0'}, 'RUN': ['5.3-latest'], '5': {'5.3': '5657'}}

File: utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -959,6 +959,10 @@ def compare_with_old_version_file(self):
                                                 help="osd-minor-version end range",
                                                 action="store", dest="osd_minor_version_end",
                                                 required=True)
+        get_latest_osd_candidate_json.add_argument("--new_run",
+                                                help="True if RUUN key is new else False",
```
Comment (tarukumar) [inline utils/scripts/ocm/ocm.py:963]: **RUUN** spel check

**Comment by tarukumar** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/interop-test/15/console
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/interop-test/16/console

**Review by tarukumar — APPROVED** [review]

**Review by aloganat — APPROVED** [review]
LGTM

**Pull request merged**